.appveyor.yml

```diff
@@ -0,0 +1,35 @@
+environment:
+  matrix:
+    - nodejs_version: Current
+      platform: x64
+
+install:
+  - ps: Install-Product node $env:nodejs_version $env:platform
+  - set CI=true
+  - npm install --global npm@latest
+  - set PATH=%APPDATA%\npm;%PATH%
+  - npm install
+  - npm install -g gulp istanbul codeclimate-test-reporter
+  - npm install ogg-packet
+
+matrix:
+  fast_finish: true
+
+platform:
+  - x64
+
+# Disable automatic builds
+build: off
```

Comment:
Does this disable builds after every commit?

Reply:
it will run on every commit. precise meaning: not sure. it's specified in default node config for appveyor so I used it here as well: https://www.appveyor.com/docs/lang/nodejs-iojs/#quick-start

Reply:
I think 'build' is used for building VS projects, not needed for node projects. Eg. check opus appveyor: https://github.com/xiph/opus/blob/master/.appveyor.yml